backend/digitales_warten.py

```diff
@@ -23,6 +18,24 @@ def create_slot(queue_id):
     db.session.commit()
     return str(slot.id)
 
+@app.route('/place/<place_id>/queues', methods=['POST'])
+def create_queue(place_id):
+    if 'application/json' not in request.headers['Content-Type']:
```

Comment:
Maybe access the header object using the associated API? https://werkzeug.palletsprojects.com/en/1.0.x/datastructures/#werkzeug.datastructures.EnvironHeaders

Reply:
No.